.npmignore

```diff
@@ -1,11 +0,0 @@
-*.log
```

Comment:
There is also [`files`](https://docs.npmjs.com/files/package.json#files) in package.json. I think we should use only one of them ‒ either `files` in package.json or .npmignore ‒ to avoid confusion (and I have no strong preference as to which one).

Reply:
You're right!

After discussing this internally, removing this file is fine indeed.

Apparently specifying what should be added through `package.json` [is the recommended way of doing it](https://blog.npmjs.org/post/165769683050/publishing-what-you-mean-to-publish).